lib/model/query/submission-attachments.js

```diff
@@ -204,6 +204,7 @@ where submission_defs.current=true
   and "deletedAt" is null
   and ${odataFilter(options.filter, odataToColumnMap)}
   and submission_attachments.name is distinct from submission_defs."encDataAttachmentName"
+  and submission_attachments."isClientAudit" is not true
```

Comment:
`isClientAudit` can be `TRUE`, `FALSE`, or `NULL`. After a quick look, the only place where I see it being set to `NULL` is for the encrypted data attachment:

https://github.com/getodk/central-backend/blob/2d8d20dd823cd478bb062d5d5b7f13070abf39f0/lib/model/query/submission-attachments.js#L28
https://github.com/getodk/central-backend/blob/2d8d20dd823cd478bb062d5d5b7f13070abf39f0/lib/model/query/submission-attachments.js#L67-L68

That attachment should be excluded by the line above this one. However, I see other submission attachments on the QA server where `isClientAudit` is `NULL`, not `FALSE`. Maybe that's just junk data, but given that the column is nullable, I think the safest thing is to include submission attachments for which `isClientAudit` is either `FALSE` or `NULL`.

That's what `IS NOT TRUE` does: https://www.postgresql.org/docs/current/functions-comparison.html. `= FALSE` or `<> TRUE` would exclude submission attachments for which `isClientAudit` is `NULL`.